### DISCUSSION TITLE
Table is not responsive when placed within a SidebarProvider

### DISCUSSION THREAD

**Discussion by demetrius-mp** [huntabyte/shadcn-svelte#1436] [Help] [answered]
### Describe the bug

When placing a table inside a SidebarProvider the overflow-x-scroll does not work

it seems the cause of this problem is the `flex` class in the `div` rendered by the SidebarProvider

### Reproduction

https://github.com/demetrius-mp/table-in-sidebar-layout

### System Info

```bash
N/A
```

### Severity

annoyance

**Comment by huntabyte** [comment]
Can you not wrap it with your own element instead of placing it directly within the flexbox?

**Reply by demetrius-mp** [reply]
i tried that, you can check the repro

seems the problem is specifically with the `flex` in the SidebarProvider

**Comment by demetrius-mp** [comment, marked as answer]
Solved by adding `overflow-hidden` to `<Sidebar.Inset />`

**Reply by huntabyte** [reply]
Awesome!
